### PR TITLE
added vconnect logo

### PR DESCRIPTION
**Pull request by tobylonge**

### CODE REVIEW

File: logos/vconnect/vconnect.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...vtOlG4G3gJ4 NZj+sz22/TkzuuCNVjouysvRJmafQpp2IjGOA2E/jPx7TxDaAPBrbOA5iNiz3rq2NXb7OxwbAjMm T6KMX4DoEgwxa7orDPwLUl7jrW9Oqk52uKrsfEG860tyr67HjshrbBuxZGIoUHKWEPQzAh2SgKoP DRY/ya377G8jHbjl4oKZrkzPTULgIgDD5ukS8KZk4wZvXcuTI9mqCfFF0M7Ac97a4CnJjFU04K+F wBfl1lgaZ3rrW/6R6Pkb506bOjEr6zPs3DxH6IpHwzNGZQC8kzVzkJE/s/AI0sQUMElh0IYurff9 RDaz7KmEFpQWa4IOhOD9mFBIRAVgzgMoF0AWAR4GS2aKESHGQBSSt5AQm5nQAoODTPFPQ73a+yqr OdhFxyUzCl0ez2yQ2B/gksHGBwC4rwd7N4AQCFuYuY2YWhjGxwbH31fV0UkVa+YgY3JR8TcEtBOJ eDYzzRCETMnoIUIbA++Rgec+peDT6XbvU0k4UHwokXYmCF8hRikIkwA2AOpgxidM9EpP17ZHza67 2hmYUaRhwlnQ5NGCaRaDfSAilhwhQUFp4HVdGn+b3Njykd1jNBRN80oyczPoTBJ0EjFmM2MaETIY 3A1QGxjvA8azb37S8ozVpbvMIFxRXCpImw0y9mUhivqubUwGkAtgQv97H4BuEHcwsBmgLYBcB8mf ktQ/+5TWN43m60A1m8unTMr0TTxIkjhAaORn8EzBtBcDuQxkE9hNINeO+2SMgG4wOkDYYjCvBzgI Q67tRc+70xo3b7Lbn1TYVDF1WoYr+yxiPpYI+wKcD5AGYCtLDoLoDakbT+QubnkrVR3hQGm5RlQh ibsNafwu2ZS5UGXJwZoQJzLhQAKKuG9zWDfAbQC9I5mfz61rehdJTmZ8fsn0KVnurDMIOJwFl3Ff p7duMDaywPswjGeT6LJL4eqyHwnm01nwut5u/vHUpc1JFT3YXD5lUkZe9q8JOEAyViQz077LWB2n aeLHDDDr+p2+xtZn/z8noQNoegS1gAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOS0wMi0xNFQwMjoy NjowMC0wNzowMI2RO1MAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTktMDItMTRUMDI6MjY6MDAtMDc6 MDD8zIPvAAAAAElFTkSuQmCC"/></svg>
```
Comment (onyekvchi) [inline logos/vconnect/vconnect.svg]: Heyy @tobylonge - thanks for contributing!
It looks like this SVG just wraps a PNG file. Can you upload a raw SVG instead?

Comment (tobylonge) [reply logos/vconnect/vconnect.svg]: Ok, I will fix, thanks for the feedback

Comment (tobylonge) [reply logos/vconnect/vconnect.svg]: Raw SVG file uploaded

**Comment by onyekvchi** [comment]
Heyy @tobylonge , please help fix the conflicts in the `js/logos.json`

**Pull request merged**